frontend/src/components/viewers/Tensorboard.tsx

```diff
@@ -56,6 +71,10 @@ class TensorboardViewer extends Viewer<TensorboardViewerProps, TensorboardViewer
     this._checkTensorboardApp();
   }
 
+  public onChangeFunc = (e: React.ChangeEvent<{ name?: string; value: unknown }>): void => {
```

Comment:
nit: give this function a meaningful name, e.g. `handleVersionSelect`

Reply:
Will do.